doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-lanes-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='bpmn-lanes-proc_{context}']
+= BPMN2 swimlanes in process designer
+
+Swimlanes are process elements that visually group tasks related to one group or user. You can use user tasks in combination with swimlanes to assign multiple user tasks to the same actor, due to `Autoclaim` property of the swimlanes. When the first task in a swimlane is created with a specified actorId, then the same actorId is also assigned to all other tasks of that swimlane. The `Autoclaim` property enables the auto-assignment of the tasks that are related to a swimlane.
```

Comment:
Is it necessary to add **``** to actorId?

Reply:
Yes, since it's a parameter that we add.

Reply:
I don't think very clear the term ActorID here. In fact, swimlanes Autoclaim functionality is about to "claim" a task from a group of potential owner usually defined by GroupId instead (the task is assigned to a group and you need to claim it, no direct ActorId is defined in the task at all). I think better to say something like: "_When the first user task in a swimlane is claimed by a potential owner from a Group, the other user tasks in that same swimlane are directly assigned to that same owner instead of remain with the Group, therefore the claim for these other tasks are not needed_." - this is why the name of the functionality is called Autoclaim.

Reply:
Done